src/libstd/io/buffered.rs

```diff
@@ -293,7 +293,7 @@ impl<R: Seek> Seek for BufReader<R> {
     /// where `n` minus the internal buffer length overflows an `i64`, two
     /// seeks will be performed instead of one. If the second seek returns
     /// `Err`, the underlying reader will be left at the same position it would
-    /// have if you seeked to `SeekFrom::Current(0)`.
+    /// have if you sought to `SeekFrom::Current(0)`.
```

Comment:
I might prefer rewording this, as "seek time" is a noun, and it's confusing if it's used as a verb like this. I agree that "seeked" is incorrect, but rewording would be preferable in my opinion (to use a different verb altogether).